<h1>Previous work</h1>

I was a <a href="http://www.mrc.ac.uk/index.htm">MRC</a> funded PhD student (2011-2014) under the supervision of <a href="http://chr1swallace.github.io/">Dr Chris Wallace</a>
from the <a href="https://www-gene.cimr.cam.ac.uk/staff/dilstats/">DIL stats group</a>, working on the analysis of Flow Cytometry data for immune-phenotype to genotype association studies in the context of Type 1 Diabetes.
I was and am still interested in normalisation and clustering.
Clusters may represent populations of cells such as in flow cytometry data, or different genotypes such as in SNP array or quantitative PCR data.
However as the level of uncertainty, data quality (e.g signal-to-noise ratio) and prior knowledge (e.g expected number of clusters), differs greatly between experimental data sets,
clustering needs to adopt a more probabilistic approach.
One of the important aspects of my work was to allow for this uncertainty in assigning elements to clusters by giving probabilistic weights of cluster membership (a number between 0 and 1)
instead of making binary calls (belongs or does not belong to cluster).  Model-based clustering (i.e. fitting a mixture of distributions) is particularly suited to this task.
The uncertainty in the clustering can be taken into account when testing for genetic association with disease or other phenotypes of interest such as protein expression or cell frequency.

<h5> Association of CD25 expression with IL2RA genotype </h5>
<p>
I started off my PhD by reanalysing, using computational methods (k-means, mixtures of univariate distributions), previous <a href="http://www.nature.com/ng/journal/v41/n9/full/ng.434.html">published data</a> to test the correlation between CD25 expression on naive and memory T cells with regulating SNPs in the IL2RA gene.
I found that automatic methods were able to outdo manual methods by improving the reproducibility of the results and the strength of the correlation but did poorly when the data did not fit
the prior assumptions (e.g. number of clusters larger than expected).  But this seeming weakness of the automatic method can be useful as it makes for an excellent outlier
detection tool, capabale of spotting cases were the data does not fit the expected model.
I summarised and presented some of my findings as a <a href="https://docs.google.com/file/d/0B2uxQDg1624kZURrYmpBUUdBVWc/edit?usp=sharing">poster (18-06-2012)</a> at the Cyto 2012 Conference in Leipzig.
</p>

<h5> Association of KIR3DL1/3DS1 with type 1 diabetes </h5>
<p>
Killer Immunoglobulin-like Receptors (KIR) play an important role in the innate immune system. Present on the surface of Natural Killer cells, KIRs mediate the fate of target cells based on the composite inhibiting/activating signal generated by binding to their HLA Class I ligands.
Type 1 Diabetes (T1D) is an autoimmune disease known to be strongly associated with the HLA region, primarily with HLA Class II genes, but also with HLA Class I loci, such as HLA-Bw4/Bw6 epitope (P=6.57E-6 conditional on HLA Class II).        Of the 17 known KIR genes, KIR3DL1 is the only one known to interact biologically with HLA-Bw4, which makes it a suitable candidate gene for T1D. Furthermore, its activating counterpart, KIR3DS1, may also putatively interact with HLA-Bw4.
We conducted a case-control study (816 cases: 813 controls) to assess whether there is evidence that copy number variation in KIR3DL1/3DS1 is associated with T1D.
<a href="https://docs.google.com/file/d/0B2uxQDg1624keXFfbEN0NERVcVE/edit?usp=sharing">Poster (28-05-2013)</a> summarising work so far.
Presented <a href="http://figshare.com/articles/COMPLEMENTING_QPCR_WITH_SNP_DATA_ACHIEVES_LARGE_SCALE_CASE_CONTROL_ANALYSIS_OF_KIR3DL1_S1_COPY_NUMBER_VARIATION_AND_REVEALS_NO_ASSOCIATION_IN_TYPE_1_DIABETES/797488">poster (07-09-2013)</a> at the Genomics of Common Disease 2013 in Oxford.
Published in <a href="http://www.biomedcentral.com/1471-2164/15/274">BMC Genomics</a>.
</p>
