<html>
<title>
Nikolas Pontikos
</title>

<p>
<iframe src="http://githubbadge.appspot.com/pontikos?s=1" style="border: 0;height: 142px;width: 200px;overflow: hidden;" frameBorder="0"></iframe>
</p>

<p>
<a href="https://uk.linkedin.com/pub/nikolas-pontikos/5/11a/401" style="text-decoration:none;"><span style="font: 80% Arial,sans-serif; color:#0783B6;"><img src="https://static.licdn.com/scds/common/u/img/webpromo/btn_in_20x15.png" width="20" height="15" alt="View Nikolas Pontikos's LinkedIn profile" style="vertical-align:middle;" border="0">&nbsp;View Nikolas Pontikos's profile</span></a>
</p>

Below is my PhD page while at University of Cambridge, I'm working on different stuff now so I will update all this when I find some time.


        <img src="https://www-gene.cimr.cam.ac.uk/staff/dilstats/Pics/niko.jpg" alt="portrait" style="float:right;margin:0 0 0 5px;" />


        Nikolas is an <a href="http://www.mrc.ac.uk/index.htm">MRC</a> funded PhD student (2011-2014) under the supervision of <a href="https://www-gene.cimr.cam.ac.uk/staff/wallace/">Chris Wallace</a>
from the <a href="https://www-gene.cimr.cam.ac.uk/staff/dilstats/">DIL stats group</a>, working on the analysis of Flow Cytometry data for immune-phenotype to genotype association studies in the context of Type 1 Diabetes.
        He holds an undergraduate MEng degree from <a href="http://www.cs.ucl.ac.uk">UCL in Computer Science</a> (2003-2007) and a postgraduate MSci in <a href="http://www.bioinformatics.imperial.ac.uk/">Bioinformatics from Imperial College</a> (2008-2009).
        He has three years of work experience as a software developer, two of which he spent at the <a href="http://www.uniprot.org">Uniprot</a> team at the <a href="http://www.ebi.ac.uk">European Bioinformatics Institute</a> (2009-2011).
        Find me at <a href="http://cambridge.academia.edu/NikolasPontikos">Academia.edu</a> and on <a href="http://www.mendeley.com/profiles/nikolas-pontikos/">Mendeley</a>.
        Also see my profile on the <a href="http://www.immunology.cam.ac.uk/directory/np385@cam.ac.uk">Cambridge Immunology page</a>.
        You can even add me on <a href="http://www.linkedin.com/in/nikolaspontikos">Linkedin</a>.
        </p>


        <div id="sub-brand"><p class="section">Research Themes</p></div>

        <p>
        <a href='http://www.immunology.cam.ac.uk/directory/research-themes/autoimmunity-inflammation'>Autoimmunity and Inflammation:</a>
        <a href='http://www.immunology.cam.ac.uk/directory/research-themes/immunogenetics'>Immunogenetics:</a>
        </p>

        <div id="sub-brand"><p class="section">Departments</p></div>

        <p>
        <a href='http://www.immunology.cam.ac.uk/departments/medical-genetics'> Department of Medical Genetics: </a> 
        <a href='http://www.immunology.cam.ac.uk/departments/cimr'> Cambridge Institute for Medical Research:  </a>
        </p>


        <div id="sub-brand"><p class="section">Supervisor</p></div>
        <p>
        <a href='http://www.immunology.cam.ac.uk/directory/chris.wallace@cimr.ac.uk'> Dr Chris Wallace </a>
        </p>


        <div id="sub-brand"><p class="section">Research Interests</p></div>

          <div id="content-primary">

        <h3>What I do</h3>
        <p>
        I work on methods of visualising and analysing biological data, in particular computationally efficient unsupervised and supervised techniques for identifying clusters.
        The purpose of clustering is to assign elements to different groups such that elements <i>within groups</i> are more similar to each other than <i>across groups</i>, in the hope that this
        grouping of data will prove useful in leading to some biological insight (<i>There is no right or wrong model, merely one that is more useful for the job at hand.</i>).
        Clusters may represent populations of cells such as in flow cytometry data, or different genotypes such as in SNP array or quantitative PCR data.
        However as the level of uncertainty, data quality (e.g signal-to-noise ratio) and prior knowledge (e.g expected number of clusters), differs greatly between experimental data sets,
        clustering needs to adopt a more probabilistic approach.
        One of the important aspects of my work is to allow for this uncertainty in assigning elements to clusters by giving probabilistic weights of cluster membership (a number between 0 and 1)
        instead of making binary calls (belongs or does not belong to cluster).  Model-based clustering (i.e. fitting a mixture of distributions) is particularly suited to this task.
        The uncertainty in the clustering can be taken into account when testing for genetic association with disease or other phenotypes of interest such as protein expression or cell frequency.
        <!--
        Statistical significance is only achieved when good signal to noise can be attained.
        Many problems in biology can be interpreted as missing data problems.
        -->
        <!--
        Striking the right balance between implementation and theory in computational statistics.
        I am particularly interested in marrying machine learning techniques and mathematical theory of statistics with the pragmatism of implementation.
        with solid theoretical assumptions
        -->

        <h3>What I've been working on</h3>

        <h5> Association of CD25 expression with IL2RA genotype </h5>
        <p>
        I started off my PhD by reanalysing, using computational methods (k-means, mixtures of univariate distributions), previous <a href="http://www.nature.com/ng/journal/v41/n9/full/ng.434.html">published data</a> to test the correlation between CD25 expression on naive and memory T cells with regulating SNPs in the IL2RA gene.
        I found that automatic methods were able to outdo manual methods by improving the reproducibility of the results and the strength of the correlation but did poorly when the data did not fit
        the prior assumptions (e.g. number of clusters larger than expected).  But this seeming weakness of the automatic method can be useful as it makes for an excellent outlier
        detection tool, capabale of spotting cases were the data does not fit the expected model.
        I summarised and presented some of my findings as a <a href="https://docs.google.com/file/d/0B2uxQDg1624kZURrYmpBUUdBVWc/edit?usp=sharing">poster (18-06-2012)</a> at the Cyto 2012 Conference in Leipzig.
        </p>

        <h5> Association of KIR with T1D </h5>
        <p>
        Killer Immunoglobulin-like Receptors (KIR) play an important role in the innate immune system. Present on the surface of Natural Killer cells, KIRs mediate the fate of target cells based on the composite inhibiting/activating signal generated by binding to their HLA Class I ligands. Type 1 Diabetes (T1D) is an autoimmune disease known to be strongly associated with the HLA region, primarily with HLA Class II genes, but also with HLA Class I loci, such as HLA-Bw4/Bw6 epitope (P=6.57E-6 conditional on HLA Class II).
        Of the 17 known KIR genes, KIR3DL1 is the only one known to interact biologically with HLA-Bw4, which makes it a suitable candidate gene for T1D. Furthermore, its activating counterpart, KIR3DS1, may also putatively interact with HLA-Bw4.
        We conducted a case-control study (816 cases: 813 controls) to assess whether there is evidence that copy number variation in KIR3DL1/3DS1 is associated with T1D.
        <a href="https://docs.google.com/file/d/0B2uxQDg1624keXFfbEN0NERVcVE/edit?usp=sharing">Poster (28-05-2013)</a> summarising work so far.
        <b>Update:</b> Presented <a href="http://figshare.com/articles/COMPLEMENTING_QPCR_WITH_SNP_DATA_ACHIEVES_LARGE_SCALE_CASE_CONTROL_ANALYSIS_OF_KIR3DL1_S1_COPY_NUMBER_VARIATION_AND_REVEALS_NO_ASSOCIATION_IN_TYPE_1_DIABETES/797488">poster (07-09-2013)</a> at the Genomics of Common Disease 2013 in Oxford.

        <!--
        <h3><a name="sec10" id="sec10"></a>Talks & Posters</h3>
        <iframe src="http://www.slideshare.net/pontikos/slideshelf" width="760px" height="570px" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:none;" allowfullscreen webkitallowfullscreen mozallowfullscreen></iframe>
        -->

        </p>
        </div>


       <div id="sub-brand"><p class="section">Publications</p></div>

       <p>
       None yet but my work on KIR is likely to lead to a publication.
       I've also got a Bioconductor package, <a href='http://www.bioconductor.org/packages/devel/bioc/html/flowBeads.html'>flowBeads</a>,
       which is to appear in Bioconductor 2.13.
       </p>


        <div id="sub-brand"><p class="section">Keywords</p></div>

        <p>
        Type 1 Diabetes, Flow Cytometry, Statistics, Machine Learning, Clustering, Data Visualisation
        </p>


        <div id="sub-brand"><p class="section">Topics</p></div>

        <p>
        Type 1 Diabetes
        </p>


<p>




</html>
